doc/installing.md

```diff
@@ -60,4 +60,10 @@ $ make install
 ```
 The default installation prefix is `/usr/local`. Change this to your liking.
 
+Other cmake variables that maybe useful to set are
+* `BOOST_ROOT` path to the root directory of Boost
+* `CMAKE_C_COMPILER` C compiler to use
+* `CMAKE_CXX_COMPILER` C++ compiler to use
```

Comment:
Documenting `CMAKE_CXX_COMPILER` has a bit more value I think, because there is hipSYCL-specific additional meaning: It is also used to set the default C++ compiler for the CPU backend when using `syclcc` later on (which can later be modified using `HIPSYCL_CPU_CXX`). Can you maybe add this information?

Reply:
done.